docs/testing/playwright.mdx

```diff
@@ -76,4 +76,172 @@ Playwright is a well-established JavaScript testing framework. This guide aims t
   ```
 </Steps>
 
+## Test Helpers
+
+The `@clerk/testing` package also provides some helper functions to sign in/sign out with Clerk in your Playwright tests without having to interact with the UI.
+To use these commands, import the `clerk` object from the `@clerk/testing/playwright` package.
+
+### `clerk.signIn`
+
+The `clerk.signIn()` function is used to sign in a user using Clerk. This helper supports only the following first factor strategies:
+
+- Password
+- Phone code
+- Email code
+
+Multi-factor authentication is not supported.
+
+> [!NOTE]
+> `clerk.signIn()` internally uses the `setupClerkTestingToken()` helper, so you don't need to call it separately.
+
+#### Prerequisites
+
+- Navigate to a non-protected page that loads Clerk via `page.goto()`, for example the index page
+
+#### Parameters
+
+`clerk.signIn()` accepts an object with the following properties:
+
+<Properties>
+  - `page`
+  - `Object`
+
+  The Playwright page object to use for the sign-in operation.
+
+  ---
+
+  - `signInParams.strategy`
+  - `'password' | 'phone_code' | 'email_code'`
+
+  The sign-in strategy. Supported strategies are:
+
+  - `password`: The command will sign in the user using the provided password and identifier.
+  - `phone_code`: You must have a user with a test phone number as an identifier (e.g., `+15555550100`).
+  - `email_code`: You must have a user with a test email as an identifier (e.g., `your_email+clerk_test@example.com`).
+
+  ---
+
+  - `signInParams.identifier`
+  - `string`
+
+  The user's identifier. This could be a username, a phone number, or an email.
+
+  ---
+
+  - `signInParams.password`
+  - `string`
+
+  The user's password. This is required only if the `strategy` is set to `'password'`.
+
+  ---
+
+  - `setupClerkTestingTokenOptions?`
+  - `Object`
+
+  Options to pass to `setupClerkTestingToken()`. Optional.
+</Properties>
```

Comment:
How should we showcase the properties here to be easier to read? The object is the following:
```ts
type PlaywrightClerkSignInParams = {
  page: Page; // playwright object
  signInParams: {
      strategy: 'phone_code' | 'email_code' | 'password';
      password?: string;
      identifier: string;
  };
  setupClerkTestingTokenOptions?: {
      frontendApiUrl?: string;
  };
};
```